### PR TITLE
Add a backslash at the end of the shell line

### PR DESCRIPTION
**Pull request by workingloong**

### CODE REVIEW

**Review by brightcoder01 — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -63,9 +63,9 @@ jobs:
           else
             docker run --rm -it \
             -e MAXCOMPUTE_PROJECT=$MAXCOMPUTE_PROJECT \
-            -e MAXCOMPUTE_AK=$ODPS_ACCESS_ID
-            -e MAXCOMPUTE_SK=$ODPS_ACCESS_KEY
-            -v $PWD:/work -w /work elasticdl:dev bash
```
Comment (terrytangyuan) [inline .travis.yml:68]: We didn't have this issue before?

Comment (wangkuiyi) [reply .travis.yml:68]: @workingloong I don't think we need `\` in `.travis.yml`. Please refer to SQLFlow's .travis.yml as an example.

**Review by wangkuiyi — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (workingloong) [reply .travis.yml:68]: We skipped the issue before I merge it from a branch of a forked repo.

**Comment by workingloong** [comment]
@wangkuiyi Here, we need "\" because we set "-|" at line 59. This flag adds "\n" at the end of each line.

**Review by wangkuiyi — APPROVED** [review]

**Review by terrytangyuan — APPROVED** [review]

**Pull request merged**